first_order_model/shrink_util.py

```diff
@@ -132,55 +132,49 @@ def set_module(mod, state_dict):
     for key1 in state_dict.keys():
         if key1 not in ['generator', 'kp_detector']:
             continue
+        if force_model is not None and key1 != force_model:
```

Comment:
what does the "force_model" attribute even mean?

Reply:
I'd name this something else that actually allows me to understand what's happening here.